concourse/tasks/delete_admin.yml

```diff
@@ -31,3 +32,6 @@ run:
       echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
       cf delete-user "${NAME}" -f
+
+      echo "Removing leaked users"
+      cf curl /v2/users | jq -r ".resources[] | .entity.username | select(. != null) | select(test(\"^${PREFIX}\"))" | xargs -r -I {} cf delete-user {} -f
```

Comment:
A note about this: cf curl /v2/users would return 50 users (see https://apidocs.cloudfoundry.org/240/users/list_all_users.html) or 100 at most (if you specify `?results-per-page=100`). It is possible that there would be no ${PREFIX} users among them albeit many ${PREFIX} users still existing in CF. In my dev deployment, it returned 228 total results, 5 total pages. Out of those, I had 7 leftover `CATS` users. When specified 100 results per page, I got 9 `CATS` users. I expect prod can have more legitimate users.

I am not sure what determines ordering of the users returned by `/v2/users` (apidocs only mention asc or desc - but not what is it ordered by), but it's possible that after several iterations of cleanup, there would be no more ${PREFIX} users returned in 1st 100 users, even though they do exist...

We can either:
* leave it as is
* extend this to return 100 users (I +1 this)
* make this do full proper pagination (response contains `next_url`) and always get all users

If we don't do full pagination, we can check by hand when removing this workaround if it has really removed all the leftover users.